README.md

```diff
@@ -87,11 +87,11 @@ from for pull requests.
 ~~~ yaml
 approvers:
     - name1
-    - name2
+    - name2 email-address or any comment
 
 reviewers:
     - name3
-    - name4
+    - name4 #any comment can follow the first word that is the gihub handle
```

Comment:
make it identical to what you write for the approvers

Reply:
ok